### PR TITLE
feat(action): add possibility to delete a card from project

### PR DESCRIPTION
**Pull request by jurijzahn8019**
Hi @alex-page 

Please check it out, i may missed something (comming from typescript)
If there is something bad, i will fix it :smile:

This could add required functionality to  #21 

Maybe delete param is bad one, then i could change it to remove: "delete" | "archive"
And implement archiving also.

Waiting for review

### CODE REVIEW

**Comment by ksvladimir** [comment]
We need this functionality too. We also need ability to add an issue to the project if it's not in the project yet, but avoid changing the column if the issue is already in the project. I would just have "action: add" to trigger such behavior - happy to submit a PR once this PR is merged.

**Comment by alex-page** [comment]
This looks great, I need to fix up some of the formatting changes.

File: src/index.js [hunk 1]
```diff
@@ -27,7 +28,11 @@ const generateMutationQuery = require('./generate-mutation-query');
 		core.debug(JSON.stringify(resource));
 
 		// A list of columns that line up with the user entered project and column
-		const mutationQueries = generateMutationQuery(resource, project, column, nodeId);
+		const mutationQueries = generateMutationQuery(resource, project, column, nodeId, action);
+		if ((action === 'delete' || action === 'archive') && mutationQueries < 1) {
```
Comment (ksvladimir) [inline src/index.js:32]: This should be `mutationQueries.length < 1`

Comment (jurijzahn8019) [reply src/index.js]: @ksvladimir Thanks! Fixed it

**Review by alex-page — APPROVED** [review]

**Comment by alex-page** [comment]
This is great thanks @jurijzahn8019. I would like to split apart archive and delete but happy to do that in a future iteration. @ksvladimir I also would love to add to a project without needing a column. I don't have the time to do this right now but feel free to create a PR.

This should now be available in `v0.5.0`.

**Comment by jurijzahn8019** [comment]
Hi @ksvladimir Check this out https://github.com/jurijzahn8019/action-is-issue-on-board#readme
Maybe this can provide a workaround for you

**Pull request merged**